### ISSUE TITLE
Merge into core?

### ISSUE THREAD

**Issue by richardjgowers** [datreant/datreant.cli#4]
Is there a reason this can't be part of the main package?

**Comment by kain88-de** [comment]
Because I didn’t update it yet with regard to core?  

There is I believe still some stuff that doesn’t work yet. Like the tag and category selection. 
We noticed that same changes to core are necessary for this library to work. 

Once everything works we can think about moving it to core.

**Comment by richardjgowers** [comment]
Sure but the reason .data is separate is to keep weighty dependencies out.  Is there a reason to keep cli out of the default/base package?

**Comment by kain88-de** [comment]
Not really. But having this single script separate at first is nice to test things. If we find it works good. Is with master we can see if we want to include it. 

Working on this made me realize that the old datreant model was not ideal and this lead to the last big refactoring introducing.

**Comment by kain88-de** [comment]
It's a really simple script now. We can think about merging it into core.  The whole thing is about 140 lines of code.

**Comment by dotsdl** [comment]
Let's PR this into `datreant` proper. I'm less concerned about dependencies these days, and `click` looks well-supported. Thanks for putting this together @kain88-de; I know it's been a long time coming!

**Comment by kain88-de** [comment]
Can we use this a bit outside of code first to see how we like it and if
the api is complete and understandable?
